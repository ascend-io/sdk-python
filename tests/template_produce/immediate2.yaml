id: immediate2
component:
  readConnector:
    container:
      immediate:
        content_some: QSxCLEMKMSwyLDNBLEIsQwo0LDUsNgo3LDgsOQ==
        object:
          - name: f1.csv
            length: 11
          - name: f2.csv
            length: 17
    updatePeriodical:
      offset: 103366736s
      period: 134217727s
