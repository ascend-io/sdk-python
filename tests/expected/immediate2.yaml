# id: immediate2
{%- set ascendImmediateDir -%}{{ resourceParentDirectory }}/immediate2{%- endset %}
{%- set ascendImmediateFiles = ["f1.csv", "f2.csv"] %}
component:
  readConnector:
    container:
      immediate: {{ construct_immediate_container(ascendImmediateDir, ascendImmediateFiles) }}
    updatePeriodical:
      offset: 103366736s
      period: 134217727s
